examples/chart/tbot/templates/_config.tpl

```diff
@@ -0,0 +1,52 @@
+{{ if not (or .Values.teleportAuthAddress .Values.teleportProxyAddress) }}
+  {{- $_ := required "`teleportAuthAddress` or `teleportProxyAddress` must be provided" "" }}
+{{ end }}
+{{ if not .Values.clusterName }}
+  {{- $_ := required "`clusterName` must be provided" "" }}
+{{ end }}
+{{ if not .Values.token }}
+  {{- $_ := required "`token` must be provided" "" }}
+{{ end }}
+{{ if and (.Values.defaultOutput.enabled) (not .Values.defaultOutput.secretName) }}
+  {{- $_ := required "`defaultOutput.secretName` must be provided" "" }}
+{{ end }}
+{{ if (and .Values.teleportAuthAddress .Values.teleportProxyAddress) }}
+  {{- $_ := required "`teleportAuthAddress` and `teleportProxyAddress` are mutually exclusive" "" }}
+{{ end }}
+{{- define "tbot.config" -}}
+version: v2
+{{- if .Values.teleportProxyAddress }}
+proxy_server: {{ .Values.teleportProxyAddress }}
+{{- end }}
+{{- if .Values.teleportAuthAddress }}
+auth_server: {{ .Values.teleportAuthAddress }}
+{{- end }}
+onboarding:
+  join_method: {{ .Values.joinMethod }}
+  token: {{ .Values.token }}
```

Comment:
I'm expecting that the first request we receive is to put this into a Kubernetes Secret.

This does not really present a security risk as yhis value is not sensitive unless the `joinMethod` is `token`, and even in this case the token is consumed on login. However, people have policies in place and will ask us to move this to be compliant, we can save the effort by offloading this to a secret and mounting it. This will also pave the road for when user will ask us to source the token from their own secret.

Reply:
Hmm yeah - I think technically we'd want to make some changes to `tbot` internals to better support that. Right now, it is possible to provide a path to a file for loading the token, so that could work. However, it might be nicer to allow embedding environment variables within the tbot config.